apps/src/studio/levels.js

```diff
@@ -2010,6 +2040,12 @@ levels.js_hoc2015_event_four_items = {
     'imageWidth': 100,
     'imageHeight': 100
   },
+
+  'progressConditions' : [
+    { required: { 'timedOut': true }, 
+      result: { 'success': false, message: msg.failedFourItemsTimeout() } }
+  ],
```

Comment:
These levels run until blocks run out due to the timeoutAfterWhenRun flag.

It would be good to bring him into it, but punting right this moment.